metricbeat/module/logstash/node/node.go

```diff
@@ -65,32 +65,18 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 // It returns the event which is then forward to the output. In case of an error, a
 // descriptive error must be returned.
 func (m *MetricSet) Fetch(r mb.ReporterV2) error {
-	if !m.MetricSet.XPack {
-		content, err := m.HTTP.FetchContent()
-		if err != nil {
-			return err
-		}
-
-		return eventMapping(r, content)
-	}
-
-	pipelinesContent, overrideClusterUUID, err := logstash.GetPipelines(m.MetricSet)
+	content, err := m.HTTP.FetchContent()
 	if err != nil {
-		m.Logger().Error(err)
-		return nil
+		return err
 	}
 
-	err = eventMappingXPack(r, m, pipelinesContent, overrideClusterUUID)
+	pipelinesContent, overrideClusterUUID, err := logstash.GetPipelines(m.MetricSet)
 	if err != nil {
-		m.Logger().Error(err)
+		return err
 	}
 
-	return nil
-}
-
-func (m *MetricSet) init() error {
-	if m.XPack {
-		return m.CheckPipelineGraphAPIsAvailable()
```

Comment:
I think we will need to call `m.CheckPipelineGraphAPIsAvailable()` every time now, right, since we plan to always call the Logstash Pipeline Graph APIs in the `Fetch()` method, no?

Reply:
I removed it because I didn't see any call to this method anywhere, haha. Anyways I guess it should be included so I added it back into the `Fetch` call directly.